Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Port_Scan.yml

```diff
@@ -3482,6 +3418,52 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "198":
+    id: "198"
+    taskid: 6a395957-07ea-45ad-9c9c-17d96ad7ee73
+    type: regular
+    task:
+      id: 6a395957-07ea-45ad-9c9c-17d96ad7ee73
+      version: -1
+      name: Check the processes reputation
+      description: Checks the file reputation of the specified hash.
+      script: '|||file'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      file:
+        complex:
+          root: alert
+          accessor: cgosha256
+          transformers:
+          - operator: append
```

Comment:
add remove empty, to ensure not passing null values to !file

Reply:
Done